src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -481,32 +500,36 @@ private void testPanelsCollapseExpand() {
         resultsPage.clickGroupByTeam();
 
         resultsPage.displayByGiverRecipientQuestion();
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-section-0-1", "ajax_auto");
         resultsPage.clickSectionCollapseStudentsButton();
         resultsPage.waitForSectionStudentPanelsToCollapse();
 
         resultsPage.displayByQuestion();
 
         ______TS("Typical case: panels expand/collapse");
 
+        assertEquals("Expand Questions", resultsPage.collapseExpandButton.getText());
+        assertEquals("Expand all panels. You can also click on the panel heading to toggle each one individually.",
+                     resultsPage.collapseExpandButton.getAttribute("data-original-title"));
+        resultsPage.verifyResultsHidden();
+
+        clickCollapseExpandButtonAndWaitForPanelsToRespond(true);
```

Comment:
I mean you have already expanded all panels here, is there a need to test it again below?